Orange/widgets/visualize/utils/plotutils.py

```diff
@@ -0,0 +1,151 @@
+from AnyQt.QtWidgets import QScrollArea, QSizePolicy, QAction
+from AnyQt.QtCore import Qt
+from AnyQt.QtGui import QKeySequence
+
+from Orange.widgets import gui
+
+
+JITTER_SIZES = [0, 0.1, 0.5, 1, 2, 3, 4, 5, 7, 10]
+
+
+def create_scroll_area(master):
+    scroll = QScrollArea(parent=master.controlArea)
+    groupbox = gui.widgetBox(scroll, "")
+    scroll.setWidget(groupbox)
+    scroll.setWidgetResizable(True)
+    master.controlArea.layout().addWidget(scroll)
+    return groupbox
+
+
+class Jittering:
+    def __init__(self, box_plot, master, jitter_sizes=None):
+        self.jitter_sizes = jitter_sizes if isinstance(jitter_sizes, list) else JITTER_SIZES
+        self.box_plot = box_plot
+        self.master = master
+
+    def __call__(self, is_cb_jitter_cont=False):
+        gui.valueSlider(
+            widget=self.box_plot, master=self.master, value='graph.jitter_size',
+            label="Jittering: ", values=self.jitter_sizes, callback=self.master.reset_graph_data,
+            labelFormat=lambda x:
+            "None" if x == 0 else ("%.1f %%" if x < 1 else "%d %%") % x)
+        if is_cb_jitter_cont:
+            gui.checkBox(
+                widget=gui.indentedBox(widget=self.box_plot),
+                master=self.master, value="graph.jitter_continuous",
+                label="Jitter numeric values", callback=self.master.reset_graph_data)
+
+
+class Box:
+    def __init__(self, master, groupbox=None):
+        self.master = master
+        self.groupbox = groupbox
+
+
+class BoxAxisData(Box):
```

Comment:
... on the other hand, I'm not sure these are proper classes. Following the current (perhaps suboptimal) practice from `gui.py`, this would be functions. They don't have any attributes except for `master` and `groupbox` that are essentially used only for passing the data to the `__call__`.

Their use is also weird: the class is constructed just to be called and then thrown away, e.g. `plotutils.BoxAxisData(self, groupbox)()`. I'd prefer `plotutils.box_axis_data(self, groupbox)`